src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
```

Comment:
Check the comment https://github.com/khshanovskyi/my-lottery-task-review/pull/1/files#r1306494992 and refactor to 
`public Ball getRandomBall() {
         int number = new Random().nextInt(100);
        return new Ball(ColorSupplier.getRandomColor(), number);
    }
`